src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static TaskList taskList;
+    private static final boolean EXIT = false;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+        boolean notExit;
+        String input = "";
+        taskList = new TaskList();
+
+        UserInterface.showsWelcomeScreen();
+
+        do {
+
+            input = scanner.nextLine();
+            notExit = checksInput(input);
+
+        } while (notExit);
+
+        UserInterface.showsExitScreen();
+    }
+
+
+    public static boolean checksInput(String input) {
+        if (input.equals(UserInterface.BYE_KW)) {
+            return EXIT;
```

Comment:
Consider using boolean values true/false instead of creating a variable to store those values.